### ISSUE TITLE
compauth=fail reason=601

### ISSUE THREAD

**Issue by pomroy** [MicrosoftDocs/OfficeDocs-o365seccomp#810]
6xx errors are not listed here and should be. I had to call Premier support to get a half-answer that 601 means that Envelope-From and From fields don't match. It can trigger a spam evaluation despite passing SPF (for SPF policy) and being DKIM signed. It fails SPF (DMARC policy version) and Microsoft triggers this "601" which is undocumented. The mechanism needs work and the whole thing needs documentation.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bab02396-69f6-5a83-fc27-7a2ba9ab78eb
* Version Independent ID: 63266ff4-ef2a-84b3-f1b9-c7b3e4d8a1b7
* Content: [Anti-spoofing protection in Office 365](https://docs.microsoft.com/en-us/office365/securitycompliance/anti-spoofing-protection)
* Content Source: [SecurityCompliance/anti-spoofing-protection.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/anti-spoofing-protection.md)
* Service: **o365-seccomp**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

**Comment by scanum** [comment]
@pomroy Thank you for submitting feedback and contributing to the docs. We are currently investigating this.
 @officedocsbot assign @scanum

**Comment by Napsty** [comment]
Any news on that? I just received news from a customer that a sent e-mail from our server was tagged as spam, with `compauth=fail reason=601`, yet 6xx reason codes are not documented as @pomroy mentions. Also SPF DNS records are set on both the `Mail From` domain and the SMTP server's domain - so is this some internal SPF/DNS lookup failure code inside O365? Without proper docs it's pretty hard to decipher this. Thanks in advance.

**Comment by garethwilliamjenkins** [comment]
I am also experiencing a similar issue I get the 601 message. When I send an email/message via the contact form (WPforms) on my wordpress website it gets sent to my junk email folder in my outlook email account. I have tried many  different fixes to no avail. If I set it to send to my gmail account instead the email/message from the contact form on my website arrives straight into my gmail inbox no problem. I have attached images of the message details from the email recieved in my junk email folder below.
[New Contact Us Today Entry.pdf](https://github.com/MicrosoftDocs/OfficeDocs-o365seccomp/files/3532260/New.Contact.Us.Today.Entry.pdf)
